client/verta/verta/_internal_utils/_utils.py

```diff
@@ -297,27 +307,6 @@ def is_workspace(self, workspace_name):
 
         return response.ok
 
-    def get_workspace_name_from_legacy_id(self, workspace_id):
```

Comment:
Just to confirm @evgeniilezhnin: Are you sure we no longer need this? For context, this was because projects and datasets used an older ID system from before WorkspaceService existed.

Are we migrating project and dataset workspace IDs to the new permissions v2 workspace IDs?

Reply:
when changing from permissions v1 to permissions v2 there is no workspace IDs format change. We changed ids format a long time ago. After that we refactored services to store resource-workspace information in UAC only in the new format. So now it should not be stored in old format anywhere and migration from old workspace ids format to new format should be already executed everywhere

Reply:
Ah, I see entities have `workspace_service_id`. Okay, this sounds good to me 👍